README.Rmd

```diff
@@ -24,7 +24,7 @@ knitr::opts_chunk$set(
 
 Antibody levels measured in a cross–sectional population sample can be translated into an estimate of the frequency with which seroconversions (infections) occur in the sampled population. In other words, the presence of many high antibody titers indicates that many individuals likely experienced infection recently and the burden of disease is high in the population, while low titers indicate a low frequency of infections in the sampled population and therefore a lower burden of disease.
 
-The **serocalculator** package was designed to use the longitudinal response characteristics using a set of modeled parameters characterizing the longitudinal response of the selected serum antibodies. More details on the underlying methods can be found in `vignette("serocalculator")` or the "Getting Started" link above.
+The **serocalculator** package was designed to use the longitudinal response characteristics using a set of modeled parameters characterizing the longitudinal response of the selected serum antibodies. More details on the underlying methods can be found in `vignette("serocalculator")` or [Getting Started](https://ucd-serg.github.io/serocalculator/articles/serocalculator.html).
```

Comment:
at least on my computer, after installing the version from your branch, `vignette("serocalculator")` doesn't actually work:

![image](https://github.com/UCD-SERG/serocalculator/assets/2474437/ff3d5fc4-eb19-4910-a369-b8a27f4ccf1b)

I think this is because you moved `serocalculator.Rmd` into the `articles` subfolder. I think the simplest solution at this point is to remove the `vignette("serocalculator")` option from this passage. The more complete solution would be to figure out how to specify the path correctly for the bibliography, but let's save that for later (please create an issue for that so we don't lose track of it as a longer-term goal).

Reply:
Ah darn. Yeah that those links and the bibliography path seem to be pretty specific. Updated the readme and will create an issue.